astrokat/observe_main.py

```diff
@@ -28,7 +28,8 @@
     for libname in libnames:
         globals()[libname] = getattr(lib, libname)
 
-TRACE = True
+global TRACE
```

Comment:
Probably don't need this `global`.  Just when modifying it in a function, like we do in `main`.